nodebuilder/share/config.go

```diff
@@ -1,42 +1,68 @@
 package share
 
 import (
-	disc "github.com/celestiaorg/celestia-node/share/availability/discovery"
+	"fmt"
+
+	"github.com/celestiaorg/celestia-node/nodebuilder/node"
+	"github.com/celestiaorg/celestia-node/share/availability/discovery"
+	"github.com/celestiaorg/celestia-node/share/availability/light"
 	"github.com/celestiaorg/celestia-node/share/p2p/peers"
 	"github.com/celestiaorg/celestia-node/share/p2p/shrexeds"
 	"github.com/celestiaorg/celestia-node/share/p2p/shrexnd"
 )
 
 type Config struct {
-	// UseShareExchange is a flag toggling the usage of shrex protocols for blocksync.
+	Availability light.Parameters `toml:",omitempty"`
+	Discovery    discovery.Parameters
+
 	UseShareExchange bool
```

Comment:
nit but i'd put this at the bottom

Also for consistency - some params are pointers and others not. Let's fix this.

Reply:
Adding this as a TODO